src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -34,4 +38,17 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
         }
         return false;
     }
+    :Override
```

Comment:
is :Override "legal"? I can't get it to compile

Reply:
> is :Override "legal"? I can't get it to compile

its ment to be @Override